src/components/FormFields/field-multi-select.tsx

```diff
@@ -0,0 +1,140 @@
+import { Fragment, useState } from "react";
+import { Listbox, Transition } from "@headlessui/react";
+import {
+  FieldValues,
+  useController,
+  UseControllerProps
+} from "react-hook-form";
+import { BsChevronExpand } from "react-icons/bs";
+import { HiCheck } from "react-icons/hi";
+
+import Label from "@/components/FormFields/box-label";
+import Error from "@/components/FormFields/error-msg";
+
+export interface FormProps<T extends FieldValues = FieldValues>
+  extends UseControllerProps<T> {
+  options: { id: number; text: string }[];
+  width?: string;
+  height?: string;
+  placeholder?: string;
+  label?: string;
+}
+
+function classNames(...classes: string[]) {
+  return classes.filter(Boolean).join(" ");
+}
+
+/*
+This is a component for a multiselect dropdown menu
+Will display a multiselect dropdown displaying options inputted
+Input:
+  options: { id: number; text: string }[]; // array of objects with option text and id number
```

Comment:
This type might be useful outside the component for defining default values. Consider declaring it as an actual type like `Option` and moving it to a separate file (or the `types.ts`),  so the single and multi select can share it